lib/utils/add_separators_util.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:flutter/material.dart';
+
+//This method returns a newly declared list with separators. It will not
+//modify the original list
+List<Widget> addSeparators(List<Widget> listOfWidgets, Widget separator) {
```

Comment:
Let's move this logic inside the Build method? Generally we need to stop using functions which returns Widget.

When we need such abstraction, we can create widget which can handle such abstraction.